apps/src/templates/Button.jsx

```diff
@@ -39,6 +39,7 @@ const ButtonHeight = {
 class Button extends React.Component {
   static propTypes = {
     className: PropTypes.string,
+    ariaLabel: PropTypes.string,
```

Comment:
I'm not sure we want to end up adding an ariaLabel to every button, considering it will be blank for most buttons. I did some reading about it [here](https://stackoverflow.com/questions/58058203/is-an-html-aria-label-attribute-allowed-to-be-an-empty-string) and am looking for more robust documentation. From a quick scan, it looks like it can make a screen reader do unpredictable things.